### PR TITLE
updated id format for revised payment instruction.

### PR DESCRIPTION
**Pull request by Lokendra-egov**

### CODE REVIEW

File: reference-adapters/ifms-adapter/src/main/resources/publicKey
Comment (shailesh-egov) [inline reference-adapters/ifms-adapter/src/main/resources/publicKey:1]: @Lokendra-egov  which public key you added ?

**Review by shailesh-egov — COMMENTED** [review]
Which public key you added?

File: reference-adapters/ifms-adapter/src/main/resources/publicKey
Comment (Lokendra-egov) [reply reference-adapters/ifms-adapter/src/main/resources/publicKey:1]: publicKey from the master branch
@shailesh-egov

Comment (Lokendra-egov) [reply reference-adapters/ifms-adapter/src/main/resources/publicKey:1]: uat

**Review by egov-arindam — APPROVED** [review]

**Pull request merged**